runtime/compiler/optimizer/J9ValuePropagation.cpp

```diff
@@ -3311,6 +3318,114 @@ J9::ValuePropagation::innerConstrainAcall(TR::Node *node)
       TR::VPConstraint *constraint = TR::VPUnresolvedClass::create(this, sig, len, owningMethod);
       addGlobalConstraint(node, constraint);
       }
+   else
+      {
+      TR::VPConstraint *constraint = createTypeHintConstraint(symRef->getOwningMethod(comp()), sig, len);
```

Comment:
As with eclipse/omr#6801, this code has started using `isUnreliableSignatureType()`. Could you please do the same adjustment here that you did in that PR? i.e. create the hint when `opaqueClass` is null *or* the type has been erased due to an interface type?

Reply:
Addressed in 53864c1